### PR TITLE
orm: add mysql

### PR DESCRIPTION
**Pull request by LouisSchmieder**
This PR adds mysql support for orm

- [x] Insert
- [x] Update
- [x] Delete
- [x] Select
- [x] Create table

### CODE REVIEW

**Comment by LouisSchmieder** [comment]
could pls someone look for opportunities for freeing stuff, because I've no clue what has to be freed

File: doc/docs.md [hunk 1]
```diff
@@ -3147,8 +3147,8 @@ fn test() []int {
 
 (This is still in an alpha state)
 
-V has a built-in ORM (object-relational mapping) which supports SQLite,
-and will soon support MySQL, Postgres, MS SQL, and Oracle.
+V has a built-in ORM (object-relational mapping) which supports SQLite and MySQL,
+but soon it will support MySQL, Postgres, MS SQL, and Oracle.
```
Comment (medvednikov) [inline doc/docs.md:3151]: mysql is already supported :)

Comment (LouisSchmieder) [reply doc/docs.md:3151]: Ah yeah typing mistake

**Comment by esquerbatua** [comment]
Pls @LouisSchmieder if you can, add a link to your docs of orm in the docs.md, something like:
For more docs about V ORM, see <a href='https://github.com/vlang/v/blob/master/vlib/orm/README.md'>vlib/orm/README.md</a>.

**Comment by esquerbatua** [comment]
ty!

**Comment by fnetX** [comment]
Hey guys, I don't really get the point: Why do I have to add `[sql: serial]` after every id field now to not break my app? What's the motivation and why can't this be skipped?

**Pull request merged**